src/assets/v3/ownership-transfer.njk

```diff
@@ -0,0 +1,28 @@
+---
+layout: layouts/base.njk
+---
+<div class="o-layout__sidebar"></div>
+
+<div class="o-layout__main o-layout-typography">
+
+<h1 id="new-chapter">Polyfill.io is entering a new chapter</h1>
+
+<p>A decade ago when polyfill.io started life, polyfills were vital to the advancement of web development practice. Today, the situation is quite different, and the time lag from a browser feature being firmly defined and it being widely supported is typically small, thanks to evergreen browsers. As a result, at the FT we aren't using polyfill.io any more, and new adoption of it has declined.</p>
+
+<p>It's time to think about what the future of polyfill.io looks like, since the FT is not going to be in a good position to be making substantial contributions in the future.</p>
+
+<p>We have therefore made the decision to transfer ownership of polyfill.io to the open source community. <a href="https://github.com/jakechampion">Jake Champion</a> who, at FT and Fastly, has been the most active contributor and maintainer for most of the polyfill.io’s life, will be taking on ownership of the service and related codebases, and will continue maintaining them with the input of other long-standing contributors. Hosting will continue to be provided free of charge by Fastly.</p>
+
+<h2 id="impact">What does this mean for you?</h2>
+
+<p>End users of the service should see no change. Transfer of ownership is largely an administrative and legal step, and the infrastructure and codebases will not be changing at all.</p>
+
+<p>We plan to transfer ownership on, or shortly after, 3rd July 2023. If you have any concerns please <a hjref="https://forms.gle/W5DGS4S7URLkTnFR9">use this form to contact us</a>. We cannot promise to respond to every enquiry but where appropriate we will share responses to common questions on this page.</p>
+
+<h2 id="alternatives">Alternatives to polyfill.io</h2>
+
+<ul>
+	<li><strong><a href="https://core-web.mrhenry.studio">core-web</a></strong>: It’s maintained by one of the maintainers of the polyfill library and provides a build-time solution for adding polyfills to your project without relying on the polyfill.io service.</li>
+	<li><strong><a href="https://babeljs.io/">babel</a></strong>: This targets features rather than browsers, but when correctly configured can be used to make sure specific features get polyfilled in your project.</li>
+</ul>
```

Comment:
Is this necessary to mention? I don't think we need to list alternatives in the ownership announcement post, the service is not going away, people don't need to find alternatives

Babel isn't an alternative to polyfills, Babel compiles new syntax to old syntax and Babel specifically states that `@babel/polyfill` "has been deprecated in favor of directly including core-js/stable (to polyfill ECMAScript features)" -- https://babeljs.io/docs/babel-polyfill

Reply:
If `polyfill.io` will remain and will continue to receive updates I also don't see the need to list alternatives.